test/integration/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/terraform-google-modules/three-tier-app/test/integration
+module github.com/GoogleCloudPlatform/terraform-google-three-tier-web-app/test/integration
```

Comment:
I've changed this because the module name should reflect the actual GitHub repo name and owner.